src/js/claims-status/utils/appeal-helpers.jsx

```diff
@@ -4,15 +4,15 @@ import moment from 'moment';
 export const hearingDescriptions = {
   video: {
     title: 'You have asked for a videoconference hearing.',
-    description: <p>You will get a letter in the mail at least 30 days before your hearing is scheduled letting you know the date, time, and location of the hearing.</p>
+    description: <p className="description-text">You will get a letter in the mail at least 30 days before your hearing is scheduled letting you know the date, time, and location of the hearing.</p>
```

Comment:
Should this class be named something different? It's really for the first paragraph in a section, maybe `claims-first-paragraph`?

Reply:
Thanks for the recommendation, that's much clearer and easier to reason about.